### DISCUSSION TITLE
We need Instant Fade.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#27514] [Ideas] [answered]
This feature currently only works in HD mode, but cannot work in other modes. In the current stable version, if the same effect needs to be achieved, the skin needs to be modified, but this method will make the hitobject lose its custom color. 

This greatly enhances the gaming experience, similar to a slider without a tail hitobject.

About the skin implementation and explanation of Instant Fade.
https://skinship.xyz/guides/insta_fade_hc
https://www.youtube.com/watch?v=jk_tNBQ9lKA
https://github.com/clarks03/Osu-Instafade-Tool

If possible, I hope to add a switch to the stable to enable or disable this function.

**Comment by Rekunan** [comment, marked as answer]
There has been discussions before:
#24018
#25473
#18789
